testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -62,13 +62,25 @@ Feature: Smoke tests for <client>
     And I enter the hostname of "<client>" as "target"
     And I click on preview
     Then I should see a "Target systems (1)" text
-    When I wait until I do not see "pending" text
-    And I click on run
+    And I wait until I do not see "pending" text
```

Comment:
When, this is an action

Reply:
I changed because in natural language those "When I wait until something happen Then I expect this other thing" it's very weird to me, it sounds better to me "Then I wait until something happen and I expect this other thing" :man_shrugging: 
BUt ok, changing, not strong opinions with those Given/When/Then/And

Reply:
@srbarrios "Then" in English has 2 meanings:
* chronology: I do this, then I do that
* cause-consequence: If I do this, then that happens

Here we use second meaning.

Reply:
> * cause-consequence: If I do this, then that happens
> 

What we write in theses steps "I wait until xxx" is:

When "I do this until that happens"

And using "Then", it sounds better to me:

Then "I do this until that happens"